packages/flame/lib/src/collisions/collision_detection.dart

```diff
@@ -161,3 +166,24 @@ abstract class CollisionDetection<T extends Hitbox<T>,
     List<RaycastResult<T>>? out,
   });
 }
+
+/// Add a basic class which implements a Listener.
+class CollisionDetectionCompletionNotifier implements Listenable {
```

Comment:
Ah, now I remember. :man_facepalming: 
I think it is probably better to go with `with ChangeNotifier` anyways, so that we don't have to do the re-implementation, and then we can just put an ignore lint lint above the overridden method.